### PR TITLE
Gitignore

### PR DESCRIPTION
**Pull request by faddat**
* adds /.vscode to the list of ignored files & paths
* improves default vscode settings

### CODE REVIEW

**Comment by assafmo** [comment]
How are these two changes working together?

**Comment by faddat** [comment]
Ah, they work fine together.

Here's how it works:

.gitignore tells git not to update those files

if we want to change the default dev env settings, then we 

git add .vscode/settings.json --force
git commit -m "update vscode settings"

File: .vscode/settings.json [hunk 1]
```diff
@@ -18,6 +18,9 @@
     "RUST_BACKTRACE": "1"
   },
   "go.useLanguageServer": true,
+  "go.lintTool": "golangci-lint",
+  "go.lintOnSave": "workspace",
+  "go.buildTags": "secretcli",
```
Comment (assafmo) [inline .vscode/settings.json:23]: So this actually messes up debug mode, since it tells it to not connect to the rust code. Can you remove this line?

**Comment by assafmo** [comment]
> git add .vscode/settings.json --force

Didn't know about this feature. Cool.

**Pull request merged**